test/SlowTests/Client/Queries/RavenDB_14541.cs

```diff
@@ -0,0 +1,394 @@
+                        query2String);
+                    Assert.Equal(2, includedDocs2.Count);
+                    Assert.Equal("Companies/App", includedDocs2.First().Key);
+                    Assert.Equal("Companies/Raven", includedDocs2.Last().Key);
+
+                    var numOfReq2 = session.Advanced.NumberOfRequests;
+                    var documents2 = session.Load<Company>(new[] { "Companies/Raven", "Companies/App" });
+                    Assert.Equal(numOfReq2, session.Advanced.NumberOfRequests);
+                }
+            }
+        }
+
+        [RavenFact(RavenTestCategory.Querying)]
+        public void IncludeWithSinglePropertyAndLet()
+        {
+            using (var store = GetDocumentStore())
+            {
+                InitializeData(store);
+
+                using (var session = store.OpenSession())
+                {
+                    var query3 = from e in session.Query<Employee>()
+                        let _ = RavenQuery.Include<Employee>(a => a.Company.Split('#', StringSplitOptions.None)[0])
+                        select e.FirstName;
+
+                    var results3String = query3.ToString();
+                    var results3 = query3.ToList();
+
+                    var includedDocs3 = ((DocumentSession)session).IncludedDocumentsById;
+                    Assert.Equal(2, includedDocs3.Count);
+                    Assert.Equal(
+                        "declare function output(e) {\r\n\tvar _ = include(e.Company.split(new RegExp(\"#\", \"g\"))[0]);\r\n\treturn {FirstName:e.FirstName};\r\n}\r\nfrom 'Employees' as e select output(e)",
+                        results3String);
+                    Assert.NotEmpty(results3);
+
+                    var numOfReq3 = session.Advanced.NumberOfRequests;
+                    var documents3 = session.Load<Company>(new[] { "Companies/Raven", "Companies/App" });
+                    Assert.Equal(numOfReq3, session.Advanced.NumberOfRequests);
+                }
+            }
+        }
+
+
+        [RavenFact(RavenTestCategory.Querying | RavenTestCategory.Indexes)]
+        public void MultipleIncludesInsideSelect()
+        {
+            using (var store = GetDocumentStore())
+            {
+                InitializeData(store);
+
+                using (var session = store.OpenSession())
+                {
+                    var query4 = session.Query<Employee, Employees_ByFirstName>()
+                        .Select(e => new
+                        {
+                            FirstName = e.FirstName,
+                            _ = Raven.Client.Documents.Queries.RavenQuery.Include<Employee>(a => a.Company.Split('#', StringSplitOptions.None)[0]),
+                            __ = Raven.Client.Documents.Queries.RavenQuery.Include<Employee>(a => a.Company.Split('#', StringSplitOptions.None)[1])
```

Comment:
is there a limit of `_` characters users can use?

Reply:
Currently, there is no limit on the '_' characters users can use. However, I can enforce a limit if needed. Let me know if you'd like me to add one, and what range or specific limit you think would be appropriate here.

I appreciate your guidance on this! :)

Reply:
All good, just making sure that there is no limit :)